lms/djangoapps/discussion/templates/discussion/discussion_profile_page.html

```diff
@@ -28,7 +29,7 @@
     profile_page_context = {
         'courseId': unicode(course.id),
         'courseName': course.display_name_with_default,
-        'userInfo': user_info,
+        'userInfo': json.dumps(user_info, default=lambda x: None),
```

Comment:
Sure. The whole `default=lambda x: none` thing is basically a guard against being passed an unserializable object somewhere in the page context tree (which would make `json.dumps` throw an error and the page not render). This behavior happens in our tests, when one of the `user_info` keys is a Python `Mock` object, which isn't serializable. So it may add additional safety to use the `default` check on the whole context object rather than just on `user_info` - but doing it just to `user_info` fixed the issue we were encountering in tests.

Do you think we should do it to the whole `profile_page_context`?